src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -382,6 +415,24 @@ protected override void TriggerAnalysis(IRequest request, IIssueConsumer consume
                     onCallSubProcess(handleMessage, request, settings, logger, cancellationToken);
                 }
             }
+
+            protected override Task<IRequest> TryCreateRequestAsync(string path, IAnalyzerOptions analyzerOptions)
```

Comment:
Overridden to provide a synchronisation mechanism so that tests can reliably wait until the factory code has completed.